docs/nuget-org/policies/deleting-packages.md

```diff
@@ -34,3 +34,27 @@ Packages that meet any of the following criteria are not allowed on the public N
 If you find a package that is in violation of any of these items, click the **Report Abuse** link on the package details page and submit a report.
 
 Note that the NuGet team and the .NET Foundation reserves the right to change these criteria at any time.
+
+#### Unlisting a package
+In order to unlist a specific package version please follow these steps:
+
+** Create a API Key **
+- Login to your Nuget.org organization account (click the "Sign in" link at the top right corner at nuget.org.
```

Comment:
Why are these instructions specific to an organization? I think they could be broadened to non-org cases as well.